lib/modules/string/kebabCase.dart

```diff
@@ -0,0 +1,8 @@
+import '_helpers/main.dart' as helpers;
+
+Function _kebabCase =
+    helpers.createCompounder((String result, String word, int index) {
+  return result + (index > 0 ? '-' : '') + word.toLowerCase();
+});
+
+String kebabCase(String string) => _kebabCase(string);
```

Comment:
Shall we document the usage of this function as in the lodash docs right above the definition here ?

Reply:
Done